### DISCUSSION TITLE
2024-10-10 CUE Community Update

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#3471] [Community Calls]
We are excited to announce our next CUE Community Update for **Thursday, Oct 10, 2024 at 1430 UTC.** 

Please reply with any questions/topics you would like to see covered.

### Agenda
- Evaluator and performance improvements
- Using CUE with other languages
- Modules & Central Registry
- cuepls and LSP
- Encodings
- JSON Schema
- Embed Proposal
- cuelang, org, docs and content strategy
- Community updates

### Meeting details
https://meet.google.com/dmy-yekv-bhc

This session will be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/swJ6y8CRKDY

### Slides
https://docs.google.com/presentation/d/1LuyRhpKMqONQnkAwHgP6EHGtdqHyEMOs4xGjRR91t2Q/

### Auto-generated transcript
https://docs.google.com/document/d/1-W4lfKKNMu5A_yfnso3Ny6kA-kIL2iopFC-YUh-1Q1I

**Comment by nxcc** [comment]
- progress on embed proposal & timeline for removing "experimental" flag

**Reply by dominikdm** [reply]
thanks, we will cover this! we will also have some time at the end of the call to answer additional questions.

**Comment by dominikdm** [comment]
Added recording, slides, and transcript

**Comment by eadlam** [comment]
This update mentioned that several lsp features would be included in the v0.11.0 release, such as go-to-definition. I installed the new vscode plugin but don't see any indication that it's connecting to the lsp. Does something need to be configured for the plugin to run the language server?

**Reply by myitcv** [reply]
@eadlam thanks for following up here. Apologies, the LSP changes didn't quite make the cut for v0.11.0. There is a large stack of changes for a new VSCode extension which are almost ready, and that will be followed by changes to `cmd/cue` which will land shortly. More information on this experimental initial version to follow!
